libs/platform/src/lib/components/list/object-list-item/object-list-item-row.component.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    Component, ElementRef,
+    ChangeDetectionStrategy, ViewEncapsulation, Input
+} from '@angular/core';
+
+@Component({
+    selector: 'fdp-object-list-item-row',
+    template: `<div class="fd-object-list__row">
+        <div class= "fd-object-list__row-left">
+            <fdp-object-attribute class="fd-object-list__object-attribute"
+             [islink]="islink" [label]="attributeLabel"
+              [linkText]="linkedText">
+              <ng-content select="#attribute"></ng-content>
```

Comment:
it is for setting special charters such as :  between text and linked text of object attribute.

Reply:
Is there a reason for using an `id` (i.e. `#attribute`) as the selector for `ng-content`? Normally HTML `id` should be unique values on a web page, and since this is in a list item it's appearance may show up multiple times.

It would be better to use a class selector (`.attribute`), or an attribute selector, e.g. `[slot='attribute']`.